### PR TITLE
[chore] Setup yarn v3.4.1

### PR DESCRIPTION
**Pull request by tomekskuta-monterail**

### CODE REVIEW

**Review by zakrzaq — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -7,3 +7,10 @@ node_modules
 .env
 dist
 .idea
+
+.yarn/*
+!.yarn/patches
+!.yarn/plugins
+!.yarn/releases
+!.yarn/sdks
+!.yarn/versions
```
Comment (WawrzenskiPatryk) [inline .gitignore:16]: new line 🤓 🚫

File: README.md [hunk 2]
```diff
@@ -14,6 +14,14 @@ asdf plugin-add nodejs
...
 ```
 
+We use Yarn v3.4.1. If you use another version of yarn type:
+
+```bash
+corepack enable
+```
+
+More details [here](https://yarnpkg.com/getting-started/install).
+
```
Comment (WawrzenskiPatryk) [inline README.md:24]: 💪

**Review by WawrzenskiPatryk — CHANGES_REQUESTED** [review]
Just one minor thing there 😄

**Review by WawrzenskiPatryk — APPROVED** [review]
LGTM 😄 ✅

**Pull request merged**